cfpb/cfgov-refresh
# Fixes for app build process

Fixes for app build process

## Changes

- Modified `scripts/npm/apps-install/index.js` to use `execSync` process to avoid race conditions with `gulp build`.  Added `process.exit` to stop build process on failure. Included apps package.*json contents in checksum frontend.sh process.

## Testing

1. Run `frontend.sh production` and verify the console content is as expected for the apps.
2. Modify the owning-a-home package.json to bump the jquery version to 5
3. Run `frontend.sh production` again, you should see an error in the console and the build should stop.


## Screenshots


## Notes

-

## Todos

-

## Checklist
